packages/rich-text/src/split.js

```diff
@@ -9,9 +9,13 @@ import { replace } from './replace';
  * split at the given separator. This is similar to `String.prototype.split`.
  * Indices are retrieved from the selection if none are provided.
  *
- * @param {Object}        value    Value to modify.
+ * @param {Object}        richText
```

Comment:
I wonder if `richText` is the right way to describe this. It seems to be called `value` everywhere at the moment.